tracer/time_windows.go

```diff
@@ -0,0 +1,31 @@
+package tracer
+
+import (
+	"golang.org/x/sys/windows"
+	"time"
+)
+
+// This method is more precise than the go1.8 time.Now on Windows
+// See https://msdn.microsoft.com/en-us/library/windows/desktop/hh706895(v=vs.85).aspx
+// It is however ~10x slower and requires Windows 8+.
```

Comment:
To me the overhead is fine only if the issue is a "real" issue. What I mean, is that I can accept having the instrumentation a bit slower (still in the order of nanoseconds) if the access to the system shared memory for the time is not optimal to users, especially if it provides bad results. It doesn't impact non-Windows users, so it's something we can try and improve later.

@LotharSee do you agree or we should tackle better this issue?